ruby/.gitignore

```diff
@@ -6,4 +6,9 @@ protobuf-jruby.iml
 target/
 pkg/
 tmp/
-tests/google/
\ No newline at end of file
+tests/google/
```

Comment:
I wasn't even aware this file existed.  Why not collapse it into the top-level one?

Reply:
Great question. I don't know what sort of exclusions belong in what level of the hierarchy. I assume that having multiple files avoids triggering overly broad reviews? If that's a desirable attribute to maintain, perhaps I should move all the `.gitignore` changes in this PR here?

Reply:
Yea that sgtm

Reply:
Done.